worker/src/RTC/Codecs/VP8.cpp

```diff
@@ -200,6 +200,12 @@ namespace RTC
 		{
 			MS_TRACE();
 
+			if (data == nullptr)
```

Comment:
If `data` was `nullptr`, which is extremely rare, near to impossible, then we would dereference a null pointer leading to undefined behaviour.